README.md

```diff
@@ -194,6 +196,19 @@ Since Xcode properly supports wrapping Swift libraries, we can just submit it fr
 
 This is only an issue if you must support iOS < 12.2.
 
+## Generating a Stack Trace (Device only)
+
+If you experience a crash on your end it's very valuable for us to have as much information as possible to provide you with the best support experience.
+Such valuable information includes a stack trace of the crash. If you don't know how to do this in Visual Studio 2019, here's a quick step-by-step guide:
+
+1. Open your terminal and enter the command `touch ~/.mtouch-launch-with-lldb`. This will essentially let you use lldb to debug your application when it launches.
```

Comment:
```suggestion
1. In the Terminal app enter the following command: `touch ~/.mtouch-launch-with-lldb`. This will essentially let you use lldb to debug your application when it launches.
```